feat(core-node)!: iota-archival removal (#11645)
iotaledger/iota: crates/iota-light-client/README.md

# A full node JSON RPC endpoint to query the latest network state (mandatory)
rpc_url: "https://api.mainnet.iota.cafe"

> # A full node GraphQL RPC endpoint to query end-of-epoch checkpoints (mandatory — used to sync end-of-epoch checkpoints)

Reviewer: ```suggestion
# An indexer GraphQL RPC endpoint to query end-of-epoch checkpoints (mandatory — used to sync end-of-epoch checkpoints)
```
